components/eam/src/physics/crm/crm_physics.F90

```diff
@@ -1611,6 +1603,14 @@ subroutine crm_physics_tend(ztodt, state, tend, ptend, pbuf2d, cam_in, cam_out,
             end do
          end do
 
+         !------------------------------------------------------------------------------------------
+         ! set convective heating tendency for gravity wave drag
+         !------------------------------------------------------------------------------------------
+         if (ttend_dp_idx > 0) then
+            call pbuf_get_field(pbuf_chunk, ttend_dp_idx, ttend_dp)
+            ttend_dp(1:ncol,1:pver) = ( ptend(c)%s(1:ncol,:pver) - qrs(1:ncol,:pver) - qrl(1:ncol,:pver) )/cpair
```

Comment:
I can see that you are subtracting radiative heating from temperature tendency. If ttend_dp is an input for gravity wave drag, is there a reason to remove radiative heating ?

Reply:
This is meant to mimic how it all works in the non-MMF cases, in which the heating tendencies are taken directly from the ZM scheme and no radiative heating is included.

Reply:
I see. thanks for clarifying it.